README.md

```diff
@@ -94,6 +94,7 @@ The following users have deployed JanusGraph in production.
 * [Netflix](https://www.netflix.com) -
   [video](https://youtu.be/KSmAdtMJYEo?t=1h2m17s) and
   [slides](https://www.slideshare.net/RoopaTangirala/polyglot-persistence-netflix-cde-meetup-90955706) (graph discussion starts at #86)
+* [RedHat](https://www.redhat.com/) - [application migration and assessment tool](https://developers.redhat.com/products/rhamt/overview/) built on [Windup](https://github.com/windup/windup)
```

Comment:
This looks good, but could you please change it to "Red Hat" (with a space between the two words)? Thanks. :)

Reply:
Fixed! Sorry about that, didn't realize it's actually two words. Will know for the future. :-)

Reply:
Almost everyone makes that mistake, due to the font. Thanks for fixing it!